### ISSUE TITLE
Mixxx 2.4.0-beta: controller duplicates

### ISSUE THREAD

**Issue by deejaygirl** [mixxxdj/mixxx#12075]
### Bug Description

Hi

Is it normal my mouse and keyboard appear under preferences -> controllers?

And if this is normal: my mouse is written 4 times there! Is it a bug?

![afbeelding](https://github.com/mixxxdj/mixxx/assets/147284581/d3423127-cecf-4e26-b19f-563f027760b5)

### Version

Mixxx 2.4.0-beta

### OS

Windows 10

**Comment by Holzhaus** [comment]
No, mice and keyboards should be filtered from the list of HID controllers IIRC.

**Comment by deejaygirl** [comment]
> No, mice and keyboards should be filtered from the list of HID controllers IIRC.

OK!
Then it's definitely a bug.
(PS: I don't have a DJ controller yet, so it's normal I don't have one listed there)

**Comment by Holzhaus** [comment]
If they still show up despite the filtering, we need them to the denylist: https://github.com/mixxxdj/mixxx/blob/main/src/controllers/hid/hiddenylist.h

**Comment by deejaygirl** [comment]
> If they still show up despite the filtering, we need them to the denylist: https://github.com/mixxxdj/mixxx/blob/main/src/controllers/hid/hiddenylist.h

I'm not a developer but if you need any information about the devices, let me know.

**Comment by mxmilkiib** [comment]
My keyboard shows up twice. How might I get what is required for the blacklist?

**Comment by mxmilkiib** [comment]
Forgot so say my two USB interfaces also show up.

Is it just the HID hex that are required?

ff:06 00:61 for the ZSA Moonlander, ff:00 00:01for the AudioQuest Dragonfly Cobalt.

**Comment by daschuer** [comment]
We maintain an deny list to hide these devices:
https://github.com/mixxxdj/mixxx/blob/main/src/controllers/hid/hiddenylist.h

**Comment by mxmilkiib** [comment]
I guess the hex codes map directly to vendor_id, product_id, usage_page, usage? What would interface_number be? I can make a PR once I've got my workflow sorted.
